main.tf

```diff
@@ -39,3 +39,8 @@ resource "google_bigquery_table" "main" {
   }
 }
 
+module "authorized_views" {
+  source           = "./view_generator"
```

Comment:
Module should be `./modules/authorized_views`.

Reply:
Done